lib/danger/danger_core/plugins/dangerfile_import_plugin.rb

```diff
@@ -36,41 +36,53 @@ def self.instance_name
     #           a danger plugin from.
     # @return [void]
 
-    def import(path)
-      raise "`import` requires a string" unless path.kind_of?(String)
-      path += ".rb" unless path.end_with?(".rb")
+    def import(path_or_url)
+      raise "`import` requires a string" unless path_or_url.kind_of?(String)
 
-      if path.start_with?("http")
-        import_url(path)
+      if path_or_url.start_with?("http")
+        import_url(path_or_url)
       else
-        import_local(path)
+        import_local(path_or_url)
       end
     end
 
-    private
-
     # @!group Plugins
-    # Download a remote plugin and use it locally.
+    # Download a local or remote plugin or Dangerfile
+    # This method will not import the file for you, use plugin.import instead
     #
-    # @param    [String] url
-    #           https URL to the Ruby file to use
-    # @return [void]
-    def import_url(url)
-      raise "URL is not https, for security reasons `danger` only supports encrypted requests" unless url.start_with?("https://")
+    # @param    [String] path
+    #           a local path or a https URL to the Ruby file to import
+    #           a danger plugin from.
+    # @return [String] The path to the downloaded Ruby file
+
+    def download(path_or_url)
+      raise "`download` requires a string" unless path_or_url.kind_of?(String)
```

Comment:
Rubocop tells us to be `kind_of?` - dunno why personally, but it's that for consistency